### PR TITLE
Added new blog post for Matplotlib figures in the IPCC SR15

### PR DESCRIPTION
**Pull request by danielhuppmann**
This blog post showcases that Matplotlib was used directly to generate many of the figures in the IPCC Special Report on Global Warming of 1.5°C, as part of a larger effort to increase openness, transparency and reproducibility of the analysis of quantitative scenarios in the report.

**Warning**: I did not manage to get the style of the blog post right without using CSS markings and captions for the figures, so I followed the advice in the [Hugo discussion forum](https://discourse.gohugo.io/t/raw-html-getting-omitted-in-0-60-0/22032) and deactivated the feature to ignore raw html. If you don't want to enable this on the entire page, please let me know how to apply the formatting without using raw html tags.

### CODE REVIEW

**Comment by story645** [comment]
Thanks so much for the contribution!

File: content/posts/ipcc-sr15/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+When preparing the SR15, the authors wanted to go beyond previous reports
+not just regarding the scientific rigor and scope of the analysis,
+but also establish new standards in terms of openness, transparency and reproducibility.
+
+The scenario ensemble was made accessible via an interactive *IAMC 1.5°C Scenario Explorer*
+([link](http://data.ene.iiasa.ac.at/iamc-1.5c-explorer/#/workspaces)) in line with the
+[FAIR principles for scientific data management and stewardship](https://www.go-fair.org/fair-principles/).
+The process for compiling, validating and analyzing the scenario ensemble
+was described in an open-access manuscript published in *Nature Climate Change*  
+(doi: [10.1038/s41558-018-0317-4](https://doi.org/)).
```
Comment (davidevaleriani) [inline content/posts/ipcc-sr15/index.md]: The link points to https://doi.org/, while it should point to https://doi.org/10.1038/s41558-018-0317-4

**Review by davidevaleriani — CHANGES_REQUESTED** [review]
This is a great post, easy to read and very interesting. Thank you so much for the contribution! Just requesting a minor change and then we publish it.

File: config.toml [hunk 2]
```diff
@@ -14,3 +14,6 @@ link1_description = "About"
 
 [taxonomies]
   category = "categories"
+
+[markup.goldmark.renderer]
+unsafe = true
```
Comment (davidevaleriani) [inline config.toml]: Please add a new line at the end of the config.toml file, and indent the unsafe like category above.

**Comment by danielhuppmann** [comment]
Thank you @davidevaleriani for the positive response and catching the broken link - I implemented the two requested  changes and double-check that all links work now.

**Pull request merged**